pkg/engine/engine.go

```diff
@@ -84,6 +88,16 @@ func NewEngine(cfg *EngineConfig) (*Engine, error) {
 	e.dirs.src = srcdir
 	e.dirs.work = workdir
 
+	// load the .env.toml file.
```

Comment:
The need for this .env.toml needs to be documented. See my suggestion at https://github.com/ipfs/testground/pull/126#issuecomment-550496063

Reply:
Done in https://github.com/ipfs/testground/pull/126/commits/a56b82c60d457a05060b71a882c8415edc63c195